src/app/pages/checkout-payment/payment-payone-ideal-detail/payment-payone-ideal-detail.component.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, Input, OnInit } from '@angular/core';
+
+import { PaymentInstrument } from 'ish-core/models/payment-instrument/payment-instrument.model';
+import { PaymentMethod } from 'ish-core/models/payment-method/payment-method.model';
+
+@Component({
+  selector: 'ish-payment-payone-ideal-detail',
+  templateUrl: './payment-payone-ideal-detail.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PaymentPayoneIdealDetailComponent implements OnInit {
+  constructor(protected cd: ChangeDetectorRef) {}
+
+  @Input() paymentMethod: PaymentMethod;
+  @Input() paymentInstrument: PaymentInstrument;
+
+  bankGroupName: string;
+
+  ngOnInit() {
+    this.bankGroupName = this.getBankGroupName();
+  }
+
+  /**
+   * function to get localized bankGroup name
+   */
+  protected getBankGroupName(): string {
+    if (this.paymentInstrument && this.paymentInstrument.parameters) {
```

Comment:
use the optional chaining operator here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
this.paymentInstrument?.parameters